aten/src/ATen/cuda/CUDAGraphsUtils.cuh

```diff
@@ -67,14 +67,18 @@ inline std::ostream& operator<<(std::ostream& os, CaptureStatus status) {
   return os;
 }
 
-inline CaptureStatus currentStreamCaptureStatus() {
-  #if defined(CUDA_VERSION) && CUDA_VERSION >= 11000
-  // don't create a context if we don't have to
-  if (at::detail::getCUDAHooks().hasPrimaryContext(c10::cuda::current_device())) {
+inline CaptureStatus currentStreamCaptureStatusMayInitCtx() {
```

Comment:
Probably would be helpful to have a little note when you should call `currentStreamCaptureStatusMayInitCtx`. IIUC, you should call this whenever it is KNOWN that the CUDA context is initialized (or initializing it would otherwise not matter) to help avoid overhead.

Reply:
done